CHANGELOG.md

```diff
@@ -39,6 +41,10 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 - Changed PPN photolysis reaction rate constant to `PHOTOL(167)`
 - Changed photolysis reactions from Travis et al (2024) rate constnats
   to`PHOTOL(168:177)`
```

Comment:
I think these lines should be removed since they are modifications that have all been made to updates added in dev/14.5.0. Also, I believe these types of changes should be included in the CHANGELOG_fullchem.md` file and not here.